### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *Description of why you made this PR*

## Solution
- *And how do you fix that problem*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wordpress_ynh%20PR109%20(Kay0u)/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by ericgaspar** [comment]
https://github.com/YunoHost-Apps/wordpress_ynh/blob/3430a30dad3f841107f042cae66eb9b65edbf385/scripts/backup#L52

Should we leave this as is?

**Comment by kay0u** [comment]
Nope :-)

File: manifest.json [hunk 1]
```diff
@@ -18,12 +18,12 @@
...
 	}],
     "requirements": {
-        "yunohost": ">= 3.6"
+        "yunohost": ">= 3.8.1"
     },
 	"multi_instance": true,
 	"services": [
 		"nginx",
-		"php7.0-fpm",
+		"php7.3-fpm",
```
Comment (ericgaspar) [inline manifest.json]: The CI is not happy with this... 
✘ [YEP-2.1?] php7.3-fpm service not installed by the install file but present in the manifest

Comment (kay0u) [reply manifest.json]: Hmm yeah probably before 4.0 the CI is not happy. Let's revert this line

Comment (kay0u) [inline manifest.json]:
```suggestion
		"php7.0-fpm",
```

**Comment by kay0u** [comment]
Bump

**Review by alexAubin — APPROVED** [review]
Super quick review, looks good to me

**Pull request merged**